modules/msk/mutect1/environment.yml

```diff
@@ -0,0 +1,9 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/environment-schema.json
```

Comment:
since conda isn't configured, we'll need to ignore the nf-test for conda in the `test.yml` file: https://github.com/mskcc-omics-workflows/modules/blob/develop/.github/workflows/test.yml#L279-L284

Reply:
a36ee1214bb9a15bfa6797c356d7dc7263f351dc